src/app/games/p4/tests/P4Rules.spec.ts

```diff
@@ -170,7 +167,10 @@ describe('P4Rules', () => {
         ];
         const state2: P4State = new P4State(board2, 12);
 
-        expect(P4Rules.getBoardValue(state1).value).toBeLessThan(P4Rules.getBoardValue(state2).value);
+        // TODO: this should actually be the opposite!
```

Comment:
Non, best veut dire "best for player who's value is positive" en fait, donc BestFor(Player.ONE), c'est confusant mais la fonction a déjà un nom sévèrement long :D

Reply:
WAAAT une fonction qui à un nom qui est l'inverse de ce qu'elle fait 1 fois sur 2, quelle horreur :dagger: (et en plus y'a aucun commentaire pour l'indiquer!) 
Donc j'ai corrigé la fonction en question, et adapté les différents appels.

Reply:
Tout le monde sais que la bonté d'un plateau c'est "à quel point le deuxième joueur gagne!" :D

Reply:
héhé x) Du coup résolu ?